Source/DebugActions.cs

```diff
@@ -630,5 +630,90 @@ private static bool IsOverrideOfAny(MethodInfo method, HashSet<MethodInfo> possi
         }
 
         #endregion
+
+        #region UseThisInstead outdated compats search
+
+        [DebugAction(CategoryName, "Search outdated mods", allowedGameStates = AllowedGameStates.Entry)]
+        private static void SearchOutdatedMods()
+        {
+            var useThisInsteadType = AccessTools.TypeByName("UseThisInstead.UseThisInstead");
+            if (useThisInsteadType == null)
+            {
+                Log.Error("Use This Instead mod is required to perform this action. It's either not present or not recognized due to an update.");
+                return;
+            }
+
+            if ((bool)AccessTools.DeclaredField(useThisInsteadType, "Scanning").GetValue(null))
+            {
+                Log.Error("Use This Instead mod is currently scanning files, please try again later.");

```

Comment:
I can confidently say it'd be better if instead of "Try later" we wait and automatically continue.